src/composables/trade/useSor.ts

```diff
@@ -619,6 +622,7 @@ export default function useSor({
           successCallback();
         }
         trackGoal(Goals.BalancerSwap);
+        if (isMainnet.value) trackGoal(Goals.BalancerSwapMainnet);
```

Comment:
may be put it in function?
`if (isMainnet.value) trackGoal(Goals.BalancerSwapMainnet);`

Reply:
https://github.com/balancer-labs/frontend-v2/pull/2456/commits/921978917f1a277be9834e99b03b4660d7c78f4b